lib/active_merchant/billing/gateways/paysafe.rb

```diff
@@ -230,55 +231,137 @@ def add_three_d_secure(post, payment, options)
         post[:authentication][:directoryServerTransactionId] = three_d_secure[:ds_transaction_id] unless payment.is_a?(String) || !mastercard?(payment)
       end
 
+      def add_cruiseline_travel_details(post, options)
+        return unless cruiseline_data = options[:cruiseline_travel_details]
+
+        post[:cruiselineTravelDetails] = {}
+        post[:cruiselineTravelDetails][:cruiseShipName] = cruiseline_data[:cruise_ship_name] if cruiseline_data[:cruise_ship_name]
+        post[:cruiselineTravelDetails][:passengerName] = cruiseline_data[:passenger_name] if cruiseline_data[:passenger_name]
+        post[:cruiselineTravelDetails][:departureDate] = cruiseline_data[:departure_date] if cruiseline_data[:departure_date]
+        post[:cruiselineTravelDetails][:returnDate] = cruiseline_data[:return_date] if cruiseline_data[:return_date]
+        post[:cruiselineTravelDetails][:country] = cruiseline_data[:country] if cruiseline_data[:country]
+        post[:cruiselineTravelDetails][:state] = cruiseline_data[:state] if cruiseline_data[:state]
+        post[:cruiselineTravelDetails][:originCity] = cruiseline_data[:origin_city] if cruiseline_data[:origin_city]
+        post[:cruiselineTravelDetails][:roomRate] = cruiseline_data[:room_rate] if cruiseline_data[:room_rate]
```

Comment:
Wondering if we should call a `.to_i` on the fields that appear to be integers in the Paysafe docs (room_rate, ticket_number, fare)